src/T2DMap.cpp

```diff
@@ -2800,28 +2800,25 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             if (!mMapViewOnly) {
                 if (selectionSize == 0) {
-                    mpCreateRoomAction = new QAction(tr("Create room", "Menu option to create a new room in the mapper"), this);
-                    mpCreateRoomAction->setToolTip(tr("Create a new room here"));
+                    mpCreateRoomAction = new QAction(tr("Create new room here", "Menu option to create a new room in the mapper"), this);
                     connect(mpCreateRoomAction.data(), &QAction::triggered, this, &T2DMap::slot_createRoom);
                     popup->addAction(mpCreateRoomAction);
                 }
 
                 if (selectionSize > 0) {
-                    auto moveRoom = new QAction(tr("Move", "2D Mapper context menu (room) item"), this);
-                    moveRoom->setToolTip(tr("Move room", "2D Mapper context menu (room) item tooltip"));
+                    auto moveRoom = new QAction(tr("Move room(s)", "2D Mapper context menu (room) item", selectionSize), this);
```

Comment:
"Room(s)" - that sounds like something that should only be in the engineering English (translation source text) - i guess we need to go to a numerus translation - which will require a **manual** update to the `./src/translations/translated/mudlet_en_US.ts` file to be done as well.
```suggestion
                    auto moveRoom = new QAction(tr("Move %n room(s)", "2D Mapper context menu (room) item", selectionSize), this);
```

Reply:
Said "manual update" involves running `lupdate -locations absolute -pluralonly ./src/ -ts ./translations/translated/mudlet_en_US.ts` from the base of the source directory and then use the `linguist` tool to fill in (and accept!) the translations (from, in this case `Move %n room(s)` to `Move %n room` and `Move %n rooms`) - for en_US you do not *have* to leave the `%n` in in the singular case - but the tool will moan slightly about it not being there...

Reply:
Thanks for reminding the en_US handling. You mention `-locations absolute` here but that is missing from https://wiki.mudlet.org/w/Translating_Mudlet#English_.28American.29_translation - was that deliberately or needs to be updated maybe?

Actually I don't want to show the %n number of rooms selected here at all, just the word "room" or "rooms" respectively.

Reply:
I don't thing that will work - consider other locales where the rules are more complicated - e..g Russian - where there is three different forms: https://doc.qt.io/qt-5/i18n-plural-rules.html - AFAICT the texts would be:
|English|Russian|
|-|-|
|Move 1 room|Переместить 1 комнату|
|Move 2 rooms|Переместить 2 комнаты|
|Move 5 rooms|Переместить 5 комнат |

Reply:
Yes I was talking about the English case, where I want to read "room" and "rooms" without number.
Other languages will obviously show other words instead, but still I want to see no numbers there at all.
Otherwise the same numbers will be shown in every other line again and again, increasing what Vadi called busy.

Did you get my question about the wiki documentation? Do you need to update it to include that parameter?